autoPyTorch/pipeline/components/training/metrics/base_metric.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Dict, Optional
+
+from pytorch_lightning.metrics.metric import Metric
+
+from sklearn.base import BaseEstimator
+
+import torch.tensor
+
+
+class autoPyTorchMetric(BaseEstimator):
```

Comment:
Do you think you can add a comment saying why you use base estimator? I think I recall you wanted to use the get params set params framework, but for future readers, It will be more understandable if there is a comment explaining that

Reply:
in the end, I just kept it to keep it consistent with the rest of the code. I dont know if we'll use get_params or set_params. I can totally change it to inherit from object.

Reply:
Yeah, any place you see the possibility to save memory, please go with it. Eventually, it will be great if we compare our ram usage to see if we have room for improvement.

Reply:
okay, will do